main.js

```diff
@@ -27,14 +27,24 @@ $(document).ready(function () {
             break;
     }
 
-    // check release time to inform people in case they are running old release
-    if (CONFIGURATOR.releaseDate > (new Date().getTime() - (86400000 * 91))) { // 1 day = 86400000 miliseconds, * 91 = 3 month window
-        console.log('Application version is valid for another: ' + Math.round((CONFIGURATOR.releaseDate - (new Date().getTime() - (86400000 * 60))) / 86400000) + ' days');
-    } else {
-        console.log('Application version expired');
-        GUI.log('You are using an old version of ' + chrome.runtime.getManifest().name + '. There may be a more recent version with improvements and fixes.');
+    // check for newer releases online to inform people in case they are running an old release
+    try {
+        var url = 'https://api.github.com/repos/betaflight/betaflight-configurator/tags';
+        $.get(url).done(function (data) {
```

Comment:
I think we have to cache te result if this in local storage limit the query to 1 per hour or so, in order to avoid triggering the GitHub's rate limiting. Otherwise this functionality (that is not all that important) will just increase the chance of the user hitting the limit when trying to load the available firmware versions.